change-beta/@azure-communication-react-f608c2c7-c0f3-4be2-98ff-78ef85c174a6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Since you are making beta changes to the API this would be a `prerelease` change,

Reply:
Ah ok, so this is just a change I can make when making a `rush changelog` command right?